examples/02_decoding/plot_haxby_full_analysis.py

```diff
@@ -79,17 +79,19 @@
     for category in categories:
         print("Processing %s %s" % (mask_name, category))
         classification_target = (stimuli[task_mask] == category)
-        mask_scores[mask_name][category] = cross_val_score(
-            classifier,
-            masked_timecourses,
-            classification_target,
-            cv=cv, scoring="roc_auc")
-
-        mask_chance_scores[mask_name][category] = cross_val_score(
-            dummy_classifier,
-            masked_timecourses,
-            classification_target,
-            cv=cv, scoring="roc_auc")
+        mask_scores[mask_name][category] = cross_val_score(classifier,
+                                                           masked_timecourses,
+                                                           classification_target,
+                                                           cv=cv.split(X=masked_timecourses, groups=session_labels),
```

Comment:
PEP8. Too long lines to read.
https://www.python.org/dev/peps/pep-0008/?#maximum-line-length

I think you should adapt to like what it was before.

Reply:
I find code clarity suffers when the arguments are indented so far from the function names, which is why I don't do that.
PEP8 suggests capping at 99 chars length instead of 79 if the long sentences are preferred, so I will shorten these to 99 chars length.
The 79 char convention was necessary when the screens were lo-res 4:3. 
FHD 16:9 & 3:2 should be comfortable with 99 chars length.

Reply:
Whatever you do flake8 should be happy given that we have one such travis environment. Normally, people don't care about this. But, I will care.